lambdas/src/types/errors.ts

```diff
@@ -47,9 +46,22 @@ export class JwtSignatureValidationError extends BaseError {
 }
 
 export class SessionNotFoundError extends BaseError {
-    constructor(id: string) {
+    constructor(public readonly id: string) {
         super(`Could not find session item with id: ${id}`);
         this.statusCode = 400; // check
```

Comment:
Understand this isn't part of your change but should we do something about this "check" comment?

Reply:
I suppose Matt, was trying to say check that this is actually equivalent in Java land, I guess he was unsure about something, I think there is a ticket to check things typescript land are equivalent in java land so this seems to fall into that space